includes/definitions/discovery/supermicro-bmc.yaml

```diff
@@ -1,74 +1,169 @@
+                    descr: 'PSU {{ $index }} Fan 1'
+                    index: 'fanRPM1.{{ $index }}'
+                    skip_values: 0
+                -
+                    oid: fanRPM2
+                    value: fanRPM2
+                    num_oid: '.1.3.6.1.4.1.21317.1.14.2.1.12.{{ $index }}'
+                    descr: 'PSU {{ $index }} Fan 2'
+                    index: 'fanRPM2.{{ $index }}'
+                    skip_values: 0
                 -
                     oid: sensorTable
                     value: sensorReading
                     num_oid: '.1.3.6.1.4.1.21317.1.3.1.2.{{ $index }}'
                     descr: sensorIDString
-                    index: sensorTableSensorNr.{{ $index }}
-                    low_limit: lcThreshold
-                    low_warn_limit: lncThreshold
-                    warn_limit: uncThreshold
-                    high_limit: ucThreshold
+                    index: 'sensorReading.{{ $index }}'
                     skip_values:
                         -
                             oid: sensorIDString
                             op: 'not_contains'
-                            value: 'FAN'
+                            value: "FAN"
+                        -
+                            oid: lcThreshold
+                            op: '='
+                            value: 0
+                    low_warn_limit: lcThreshold
+                    warn_limit: ucThreshold
+                    low_limit: lnrThreshold
+                    high_limit: unrThreshold
         voltage:
             data:
+                -
+                    oid: inputVoltage
+                    value: inputVoltage
+                    num_oid: '.1.3.6.1.4.1.21317.1.14.2.1.3.{{ $index }}'
+                    descr: 'PSU {{ $index }} Input Voltage'
+                    index: 'inputVoltage.{{ $index }}'
                 -
                     oid: sensorTable
                     value: sensorReading
                     num_oid: '.1.3.6.1.4.1.21317.1.3.1.2.{{ $index }}'
                     descr: sensorIDString
-                    index: sensorTableSensorNr.{{ $index }}
-                    low_limit: lcThreshold
-                    low_warn_limit: lncThreshold
-                    warn_limit: uncThreshold
-                    high_limit: ucThreshold
+                    index: 'sensorReading.{{ $index }}'
                     skip_values:
                         -
                             oid: sensorIDString
-                            op: 'not_in_array'
-                            value:
-                                - Vcpu
-                                - VDIMM
-                                - 12V
-                                - 5VCC
-                                - 3.3VCC
-                                - VBAT
-                                - AVCC
-                                - VSB
+                            op: 'not_contains'
+                            value: "V"
```

Comment:
This is not working, from the testdata:
```
1.3.6.1.4.1.21317.1.3.1.13.47|4|12V
1.3.6.1.4.1.21317.1.3.1.13.48|4|5VCC
1.3.6.1.4.1.21317.1.3.1.13.49|4|3.3VCC
1.3.6.1.4.1.21317.1.3.1.13.50|4|VBAT
1.3.6.1.4.1.21317.1.3.1.13.51|4|Vcpu1
1.3.6.1.4.1.21317.1.3.1.13.52|4|Vcpu2
1.3.6.1.4.1.21317.1.3.1.13.53|4|VDimmP1ABC
1.3.6.1.4.1.21317.1.3.1.13.54|4|VDimmP1DEF
1.3.6.1.4.1.21317.1.3.1.13.55|4|VDimmP2ABC
1.3.6.1.4.1.21317.1.3.1.13.56|4|VDimmP2DEF
1.3.6.1.4.1.21317.1.3.1.13.57|4|12VSB
1.3.6.1.4.1.21317.1.3.1.13.58|4|3.3VSB
1.3.6.1.4.1.21317.1.3.1.13.59|4|P1V8_PCH
1.3.6.1.4.1.21317.1.3.1.13.60|4|PVNN_PCH
1.3.6.1.4.1.21317.1.3.1.13.61|4|P1V05_PCH
1.3.6.1.4.1.21317.1.3.1.13.62|4|Chassis Intru
1.3.6.1.4.1.21317.1.3.1.13.63|4|PS1 Status
1.3.6.1.4.1.21317.1.3.1.13.64|4|PS2 Status
1.3.6.1.4.1.21317.1.3.1.13.65|4|NVMe_SSD Temp  
```

This would match for example:
> 1.3.6.1.4.1.21317.1.3.1.13.65|4|NVMe_SSD Temp

Reply:
Oh I see now that Temp matches below